fluent/fluent.go

```diff
@@ -337,7 +338,7 @@ type gRIBIModify struct {
 }
 
 // AddEntry creates an operation adding the set of entries specified to the server.
-func (g *gRIBIModify) AddEntry(t testing.TB, entries ...gRIBIEntry) *gRIBIModify {
+func (g *gRIBIModify) AddEntry(t testing.TB, entries ...GRIBIEntry) *gRIBIModify {
```

Comment:
GRIBIEntry is an interface. Do you need this exposed so you can implement this interface in the test? The fluent API is intending you to use the IPV4Entry() etc methods to create the object you wish to pass to AddEntry. Maybe I want to understand this change better. Will ping you offline.